cmd/module/cmd.go

```diff
@@ -26,7 +26,7 @@ func realMain() error {
 		return err
 	}
 
-	err = myMod.AddModelFromRegistry(ctx, camera.API, viamrtsp.ModelH264)
+	err = myMod.AddModelFromRegistry(ctx, camera.API, viamrtsp.Model)
```

Comment:
can we add the other as an alias or something so as not to break existing users?

Reply:
Reverted model name back to `rtsp-h264`.

Reply:
Now it's a lie :) 
Can't we have 2 names for the same thing for now

Reply:
We can @seanavery you'd need to register:

```var Model = family.WithModel("rtsp")``` in rtsp.go now, and add a new 

`Register` function to init with the same constructor as the `ModelH264`, and add a
```err = myMod.AddModelFromRegistry(ctx, camera.API, viamrtsp.Model)```

It's a little off kilter from how we currently register different models in one module but should work.